tutorials/shaders/compute_shaders.rst

```diff
@@ -0,0 +1,245 @@
+.. _doc_compute_shaders:
+
+Using compute shaders
+=====================
+
+
+Unlike most shaders, compute shaders are independent from the graphics pipeline and do not have a fixed-functionality.
+We can think of them as blocks of code executed on the GPU for any purpose we want. Contrast this with
+fragment shaders which are used specifically for assigning a color to a fragment in a render target.
+The big benefit of compute shaders over code executed on a CPU is the high amount of parallelization that GPUs provide.
+
+Because compute shaders are independent of the graphics pipeline we don't have any user defined inputs or outputs (like a mesh going into the vertex shader or a texture coming out of a fragment shader). Instead compute shaders make changes directly on the GPUs memory which we can read and write  using scripts.
```

Comment:
```suggestion
Because compute shaders are independent of the graphics pipeline we don't have any user defined inputs or outputs
(like a mesh going into the vertex shader or a texture coming out of a fragment shader). Instead, compute shaders
make changes directly to memory stored on the GPU from which we can read and write using scripts.
```